DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/main.cpp

```diff
@@ -14,7 +14,7 @@
 // e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
 #include "dpc_common.hpp"
 
-// include the merge sort kernel header
+#include "constexpr_math.hpp"
```

Comment:
I like that you commented in merge_sort.hpp where the shared files can be found, I suggest we always do this.  When it's just a single file you can add the comment on the same line as the #include.